src/docs/issue-platform-detectors.mdx

```diff
@@ -0,0 +1,128 @@
+    - Transaction name: Considering the transaction name associated with the query, noting that different transactions can exhibit the same N+1 problem.
+    - Tags: Exploring any relevant tags associated with the query.
+    - SDKs: Taking into account the SDKs type or version in the transaction.
+- Unique aspects relevant to the problem:
+    - Problem Class: Capturing the specific problem class of N+1 database queries.
+    - Repeated span operation and description: Identifying the repeated span operation and description that characterizes the issue.
+    - Span that initiated repeated spans (preceding span): Analyzing the span responsible for initiating the occurrence of repeated spans.
+    - Parent span of repeated and preceding spans: Considering the parent span that encompasses both the repeated and preceding spans.
+    - Transaction name: Recognizing that different transactions can manifest the same N+1 problem.
+    - Tags: Taking into account any tags associated with the query.
+    - Any dynamic fields: Considering the presence of dynamic fields within the query.
+- Creation of resulting fingerprint:
+    - Problem Class: Incorporating the problem class of N+1 database queries into the fingerprint.
+    - Parameterized repeated span description: Using parameterized descriptions of the repeated span in the fingerprint.
+    - Parameterized source span description: Including parameterized descriptions of the source span in the fingerprint.
+    - Parameterized parent span description + parent operation: Incorporating parameterized descriptions of the parent span, along with the parent operation, in the fingerprint.
+
+By utilizing this fingerprinting approach, we can effectively identify and classify N+1 database queries based on their unique characteristics and contextual information, ensuring effective grouping.
+
+# Actionability/Usefulness
+
+Once you have figured out the fingerprinting process and know how to group instances of the same problem with the same fix, the next step is to provide users with all the necessary information to fix the issue. The quality of an issue lies not only in its accurate detection but also in its actionability.
+
+ N+1 Database queries heavily rely on span analysis to identify problematic spans and guide users toward a potential fix.
+
+Reporting circumstantial data is not useful! This is a crucial distinction between issues and alerts. Issues should pinpoint root problems and help resolve them.
+![Untitled](img/issuePlatformFingerprinting01.jpg)
+
+# Signal vs. Noise
+
+With every new issue type, we could be adding more and more noise to the Issues product. New issues fatigue is real for our customers. This is why it’s important to understand how a new detector you want to ship affects the overall noise of the Issue Stream.
+
+Accurate detection of truly valuable problems and correct grouping can transform "noise" into "signal." Every detector should be evaluated against this standard.
```

Comment:
Should talk about # occurrences for the noise gate since it'll help people if they don't know you can reduce the noise that way.

Reply:
nice totally. it's already mentioned [here](https://develop.sentry.dev/issue-platform/#register-an-issue-type) but connecting them together would be good